_pages/sponsors.md

```diff
@@ -10,34 +10,41 @@ jumbotron:
   background-image: /assets/images/content/bkk19-backpack-giveaway.jpg
   buttons: []
   carousel-images: ""
-  title: "Thank you to our Linaro Virtual Connect 2021 Sponsors "
+  title: Sponsorship
   title-class: big-title
-  description: |+
-    Gold sponsors: Arm, Huawei, Qualcomm
-    Bronze sponsors: Avnet, Xilinx 
-
+  description: ""
   slider:
     - image: /assets/images/content/many_screens.jpg
-      title: "Thank you to our Linaro Virtual Connect 2021 Sponsors "
-      description: >-
-        Gold sponsors: Arm, Huawei, Qualcomm
-
-        Bronze sponsors: Avnet, Xilinx 
-
-
-
-        Sponsorship opportunities for the next Linaro Virtual Connect will be announced in May 2021.
+      title: " "
+      description: "\n"
       darken: true
 flow:
   - row: container_row
     sections:
       - format: feature_block
         feature_block_content:
           position: left
-          text: |-
-            Gold sponsors: Arm, Huawei, Qualcomm
-            Bronze sponsors: Avnet, Xilinx 
-          title: Thank you to our Linaro Virtual Connect 2021 Sponsors!
+          text: "\n"
+          title: Sponsorship Information
           type: image
-          image_content_path: /assets/images/content/lvc21_sponsors.jpg
+          image_content_path: ""
```

Comment:
@kristinedill - if you're using this `feature_block` section, you'll need and image. Jekyll is failing to build the site since this image path is missing.

Reply:
Thanks, Kyle. I'll get a new image and try adding it.